src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -36,63 +36,65 @@ public class CommandTestUtil {
             put("telegramHandle", "@linusrichards");
         }};
 
-    public static final String VALID_NAME_AMY = "Amy Bee";
-    public static final String VALID_NAME_BOB = "Bart Choo";
-    public static final String VALID_PHONE_AMY = "11111111";
-    public static final String VALID_PHONE_BOB = "22222222";
-    public static final String VALID_EMAIL_AMY = "amy@example.com";
-    public static final String VALID_EMAIL_BOB = "bob@example.com";
-    public static final String VALID_ADDRESS_AMY = "Block 312, Amy Street 1";
-    public static final String VALID_ADDRESS_BOB = "Block 123, Bobby Street 3";
-    public static final String VALID_TELEGRAM_HANDLE_AMY = "@amyfarrahFowler";
-    public static final String VALID_TELEGRAM_HANDLE_BOB = "@BobLovesAliceButHatesEve";
-    public static final String VALID_TAG_HUSBAND = "husband";
-    public static final String VALID_TAG_FRIEND = "friend";
-    public static final String VALID_TAG_MODULE = "CS1101S";
-    public static final String VALID_TAG_MODULE1 = "GEA1000";
-
-    public static final String NAME_DESC_AMY = " " + PREFIX_NAME + VALID_NAME_AMY;
-    public static final String NAME_DESC_BOB = " " + PREFIX_NAME + VALID_NAME_BOB;
-    public static final String PHONE_DESC_AMY = " " + PREFIX_PHONE + VALID_PHONE_AMY;
-    public static final String PHONE_DESC_BOB = " " + PREFIX_PHONE + VALID_PHONE_BOB;
-    public static final String EMAIL_DESC_AMY = " " + PREFIX_EMAIL + VALID_EMAIL_AMY;
-    public static final String EMAIL_DESC_BOB = " " + PREFIX_EMAIL + VALID_EMAIL_BOB;
-    public static final String TELEGRAM_DESC_AMY = " " + PREFIX_TELEGRAM_HANDLE + VALID_TELEGRAM_HANDLE_AMY;
-    public static final String TELEGRAM_DESC_BOB = " " + PREFIX_TELEGRAM_HANDLE + VALID_TELEGRAM_HANDLE_BOB;
-
-    public static final String ADDRESS_DESC_AMY = " " + PREFIX_ADDRESS + VALID_ADDRESS_AMY;
-    public static final String ADDRESS_DESC_BOB = " " + PREFIX_ADDRESS + VALID_ADDRESS_BOB;
-    public static final String TAG_DESC_FRIEND = " " + PREFIX_GROUP_TAG + VALID_TAG_FRIEND;
-    public static final String TAG_DESC_HUSBAND = " " + PREFIX_GROUP_TAG + VALID_TAG_HUSBAND;
-    public static final String TAG_DESC_MODULE = " " + PREFIX_MODULE_TAG + VALID_TAG_MODULE;
-    public static final String TAG_DESC_MODULE1 = " " + PREFIX_MODULE_TAG + VALID_TAG_MODULE1;
+    public static final String NAME_ALEX = "Alex Quentin";
+    public static final String NAME_BEN = "Bart Choo";
```

Comment:
We have to standardise the name, it might raise confusion as to whether the person in question here is BEN or BART.

Reply:
Should we just settle on one person named "A" and one person named "B"?